### PR TITLE
Change the crontab samples in cronjob commands

### PR DESCRIPTION
**Pull request by daisy-ycguo**
## Proposed Changes

* Change the crontab string samples in the cronjob commands to avoid confusion. Refer to [crontab editor](https://crontab.guru/#*_*_*_*_*/2) to see the explanation.

### CODE REVIEW

File: docs/cmd/kn_source_cronjob_create.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ kn source cronjob create NAME --schedule SCHEDULE --sink SINK --data DATA [flags
 ```
 
   # Create a crontab scheduler 'my-cron-trigger' which fires every minute and sends 'ping' to service 'mysvc' as a cloudevent
-  kn source cronjob create my-cron-trigger --schedule "* * * * */1" --data "ping" --sink svc:mysvc
+  kn source cronjob create my-cron-trigger --schedule "*/1 * * * *" --data "ping" --sink svc:mysvc
```
Comment (rhuss) [inline docs/cmd/kn_source_cronjob_create.md]:
```suggestion
  kn source cronjob create my-cron-trigger --schedule "* * * * *" --data "ping" --sink svc:mysvc
```

You can totally omit the "/1" as this is the default anyway.

File: docs/cmd/kn_source_cronjob_update.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ kn source cronjob update NAME --schedule SCHEDULE --sink SERVICE --data DATA [fl
 ```
 
   # Update the schedule of a crontab source 'my-cron-trigger' to fire every minute
-  kn source cronjob update my-cron-trigger --schedule "* * * * */1"
+  kn source cronjob update my-cron-trigger --schedule "*/1 * * * *"
```
Comment (rhuss) [inline docs/cmd/kn_source_cronjob_update.md]:
```suggestion
  kn source cronjob update my-cron-trigger --schedule "* * * * *"
```

File: pkg/kn/commands/source/cronjob/create.go [hunk 3]
```diff
@@ -35,7 +35,7 @@ func NewCronJobCreateCommand(p *commands.KnParams) *cobra.Command {
 		Short: "Create a CronJob source.",
 		Example: `
   # Create a crontab scheduler 'my-cron-trigger' which fires every minute and sends 'ping' to service 'mysvc' as a cloudevent
-  kn source cronjob create my-cron-trigger --schedule "* * * * */1" --data "ping" --sink svc:mysvc`,
+  kn source cronjob create my-cron-trigger --schedule "*/1 * * * *" --data "ping" --sink svc:mysvc`,
```
Comment (rhuss) [inline pkg/kn/commands/source/cronjob/create.go]:
```suggestion
  kn source cronjob create my-cron-trigger --schedule "* * * * *" --data "ping" --sink svc:mysvc`,
```

File: pkg/kn/commands/source/cronjob/update.go [hunk 4]
```diff
@@ -35,7 +35,7 @@ func NewCronJobUpdateCommand(p *commands.KnParams) *cobra.Command {
 		Short: "Update a CronJob source.",
 		Example: `
   # Update the schedule of a crontab source 'my-cron-trigger' to fire every minute
-  kn source cronjob update my-cron-trigger --schedule "* * * * */1"`,
+  kn source cronjob update my-cron-trigger --schedule "*/1 * * * *"`,
```
Comment (rhuss) [inline pkg/kn/commands/source/cronjob/update.go]:
```suggestion
  kn source cronjob update my-cron-trigger --schedule "* * * * *"`,
```

Comment (daisy-ycguo) [reply pkg/kn/commands/source/cronjob/update.go]: done

File: pkg/kn/commands/source/cronjob/create.go (hunk 3 above)
Comment (daisy-ycguo) [reply pkg/kn/commands/source/cronjob/create.go]: done

File: docs/cmd/kn_source_cronjob_update.md (hunk 2 above)
Comment (daisy-ycguo) [reply docs/cmd/kn_source_cronjob_update.md]: done

File: docs/cmd/kn_source_cronjob_create.md (hunk 1 above)
Comment (daisy-ycguo) [reply docs/cmd/kn_source_cronjob_create.md]: done

**Comment by rhuss** [comment]
thanks !

/lgtm

**Comment by rhuss** [comment]
/hold

This will unfortunately change completely when moving to a ping source.

**Comment by rhuss** [comment]
@daisy-ycguo Since we changed now to the PingSource and removed the CronJobSource this PR cannot be applied anymore. I tried my best to add all your changes in the PingSource docs. Please open a new PR if I something missed. In the meantime, I'm closing this PR.

**Pull request closed without merging**